src/bin.js

```diff
@@ -2,4 +2,9 @@
 
 'use strict'
 
-require('./')()
+// First param is the target version
+// Second param is the target platform
+// Third param is the target architecture
+require('./')(process.argv[2], process.argv[3], process.argv[4])
+  .then(() => process.exit(0))
+  .catch((e) => process.exit(1))
```

Comment:
The errors (all output) are in index.js / error(). They get output, see the tests.